apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/pensjonforvalter/PensjonforvalterClient.java

```diff
@@ -133,7 +133,7 @@ private void lagreTpForhold(PensjonData pensjonData, DollyPerson dollyPerson, Se
                             var forholdResponse = pensjonforvalterConsumer.lagreTpForhold(lagreTpForholdRequest);
                             mergePensjonforvalterResponses(forholdResponse, response);
 
-                            if (nonNull(tp.getYtelser())) {
+                            if (nonNull(tp.getYtelser()) && !tp.getYtelser().isEmpty()) {
```

Comment:
Du kan evt unngå den første null-sjekken her hvis du i PensjonData legger til gettere for collection a la:
```
public List<TpOrdning>getTp() {
    if (isNull(tp)) {
       tp = new ArrayList();
    }
   return tp;
}
```
Tilsvarende for ytelser.

Reply:
👍  ja, det liker jeg enda bedre.